public/js/extra/chart-pie.js

```diff
@@ -0,0 +1,69 @@
+// Set new default font family and font color to mimic Bootstrap's default styling
+Chart.defaults.global.defaultFontFamily = 'Nunito', '-apple-system,system-ui,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,sans-serif';
+Chart.defaults.global.defaultFontColor = '#858796';
+
+// Pie Chart
+$(document).ready(function () {
+  $.ajax({
+    url: "/library/dataPie.php",
+    method: "GET",
+    success: function (dataPie) {
+      var name = [];
+      var percent = [];
+      dataPie = JSON.parse(dataPie)
+      for (var i in dataPie) {
+        name.push("" + dataPie[i].name);
+        percent.push(dataPie[i].posts_count);
+      }
+
+      var chartdata = {
+        labels: name,
+        datasets: [
+          {
+            label: "Posts",
+            data: percent,
+            backgroundColor: ['#ffb032', '#4e73df', '#1cc88a', '#36b9cc'],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Mamoon97150/Blog-PHP/pullRequest?prid=7234961)
```suggestion
            backgroundColor: ["#ffb032", '#4e73df', '#1cc88a', '#36b9cc'],
```